src/shared/components/community/communities.tsx

```diff
@@ -145,57 +145,41 @@ export class Communities extends Component<any, CommunitiesState> {
                   {this.state.listCommunitiesResponse.data.communities.map(
                     cv => (
                       <tr key={cv.community.id}>
-                        <td>
+                        <td className="text-start">
                           <CommunityLink community={cv.community} />
                         </td>
-                        <td className="text-right">
-                          {numToSI(cv.counts.subscribers)}
-                        </td>
-                        <td className="text-right">
-                          {numToSI(cv.counts.users_active_month)}
-                        </td>
-                        <td className="text-right d-none d-lg-table-cell">
+                        <td>{numToSI(cv.counts.subscribers)}</td>
```

Comment:
Why remove `text-right`? This was intentional so numbers are lined up

Reply:
I can make them right aligned again, but `text-right` isn't a valid bootstrap class in v5 regardless.

Reply:
Oh ha you're right -- I had to do this in #1538 too